tests/testSchema.py

```diff
@@ -103,7 +104,7 @@ def testKey(name, key):
         self.assertEqual(keys[3].getBit(), 1)
         for n1, n2 in zip(schema4.getOrderedNames(), "abcd"):
             self.assertEqual(n1, n2)
-        keys2 = map(lambda x: x.key, schema4.asList())
+        keys2 = [x.key for x in schema4.asList()]
```

Comment:
`__iter__` actually delegates to `asList`, due to Swig-related implementation details.